src/chocolatey/AssemblyExtensions.cs

```diff
@@ -73,14 +73,14 @@ public static Stream get_manifest_stream(this IAssembly assembly, string manifes
         /// <param name="assembly">The assembly.</param>
         /// <returns></returns>
         /// <remarks>Borrowed heavily from http://dhvik.blogspot.com/2009/05/assemblynamegetpublickeytoken-tostring.html </remarks>
-        public static string get_public_key_token(this IAssembly assembly)
+        public static string GetPublicKeyToken(this IAssembly assembly)
         {
             if (assembly == null) return string.Empty;
 
-            return assembly.GetName().get_public_key_token();
+            return GetPublicKeyTokenString(assembly.GetName());

```

Comment:
Hmm... oh, I think I did this initially because there was a naming conflict, this was originally going to just be GetPublicKeyToken and Assembly already has a public method with that name. Can update this now that I resolved that, I think.